CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCSourcePane.java

```diff
@@ -309,9 +315,9 @@ public JDBCSourcePane(final JDBCImportController importController) {
 
             final Button removeBtn1 = new Button("Remove");
             removeBtn1.setOnAction(removebtnAction);
-            driversTabGridPane.add(removeBtn1, 2, 1, 1, 1);
+            driversTabGridPane.add(removeBtn1, 1, 1, 1, 1);
 
-            final Button buttonCancel = new Button(ACTION_CANCEL);
+            final Button buttonCancel = new Button("OK");
```

Comment:
same here

Reply:
@Atlas139mkm can you rename this to `buttonOk` or something better?